test/adapter.spec.js

```diff
@@ -262,8 +270,8 @@ describe('jasmine adapter', function () {
         expect(result.time).toBe(1) // 4 - 3
       })
 
-      reporter.specStarted(spec.result)
-      reporter.specDone(spec.result)
+      reporter.specStarted(copySpecResult())
```

Comment:
or maybe `reporter.specStarted(spec.result.slice(0))`

Reply:
How will that work? `spec.result` is not an `array` type: `TypeError: spec.result.slice is not a function`.

We could do something like `JSON.parse(JSON.stringify(spec.result))` (or an other way of cloning the result) if you prefer.

Reply:
How about
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign

Reply:
Awesome! I've changed it:

```js
      reporter.specStarted(Object.assign({}, spec.result))
      reporter.specDone(Object.assign({}, spec.result))
```